examples/js/loaders/GLTF2Loader.js

```diff
@@ -385,6 +392,24 @@ THREE.GLTF2Loader = ( function () {
 
 	}
 
+	/* DRACO MESH COMPRESSION EXTENSION */
+
+	function GLTFDracoMeshCompressionExtension () {
+
+		this.name = EXTENSIONS.KHR_DRACO_MESH_COMPRESSION;
+
+		this.dracoLoader = new THREE.DRACOLoader();
+
+	}
+
+	GLTFDracoMeshCompressionExtension.prototype.decodePrimitive = function ( primitive, dependencies ) {
+
+		var bufferViewID = primitive.extensions[ this.name ].bufferView;
```

Comment:
According to the conformance, we need to use the `version` property to verify if the version of the encoded mesh and decoder are compatible, e.g. > 0.9.1.
@ondys
@FrankGalligan